corehq/apps/reports/v2/filters/case_columns.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy
+
+from corehq.apps.es import queries
+from corehq.apps.reports.v2.models import (
+    BaseColumnFilter,
+    FilterChoiceMeta,
+)
+
+
+class TextCaseColumnFilter(BaseColumnFilter):
+    filter_type = 'case_text'
+    title = ugettext_lazy("Text")
+    choices = [
+        FilterChoiceMeta('contains', ugettext_lazy('Contains')),
+        FilterChoiceMeta('equals', ugettext_lazy('Equals')),
+        FilterChoiceMeta('not_contains', ugettext_lazy('Does Not Contain')),
+        FilterChoiceMeta('not_equals', ugettext_lazy('Does Not Equal')),
+    ]
+
+    def get_filtered_query(self, query, config):
+        property_name = config['propertyName']
+        filter_name = config['filterName']
```

Comment:
Calling this a `name` is confusing. I think this is an `operation` or `operator`. That way you have `{property_name} {operator} {value}` which is what this method is doing.

(I think this should be changed everywhere in this PR)